<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtClMainWindow</class>
 <widget class="QMainWindow" name="mdtClMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbEditUnit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit units</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEditArticle">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit articles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEditVehicleType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>137</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit vehicle types</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actOpenDatabase"/>
    <addaction name="actCloseDatabase"/>
    <addaction name="actCreateNewDatabase"/>
    <addaction name="actImportDatabase"/>
    <addaction name="separator"/>
    <addaction name="ActQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edition">
    <property name="title">
     <string>&amp;Edition</string>
    </property>
    <addaction name="actEditVehicleType"/>
    <addaction name="separator"/>
    <addaction name="actEditConnector"/>
    <addaction name="separator"/>
    <addaction name="actEditArticle"/>
    <addaction name="separator"/>
    <addaction name="actEditUnit"/>
    <addaction name="separator"/>
    <addaction name="actEditLinkBeam"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actEditTestNode"/>
    <addaction name="actEditTest"/>
    <addaction name="actEditTestItem"/>
    <addaction name="actEditTestConnectionCable"/>
    <addaction name="actCableChecker"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edition"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDatabase"/>
  </widget>
  <action name="actOpenDatabase">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>&amp;Open a database</string>
   </property>
  </action>
  <action name="actCreateNewDatabase">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>Create a &amp;new database</string>
   </property>
  </action>
  <action name="actImportDatabase">
   <property name="text">
    <string>&amp;Import a database</string>
   </property>
  </action>
  <action name="ActQuit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actCloseDatabase">
   <property name="text">
    <string>&amp;Close database</string>
   </property>
  </action>
  <action name="actEditVehicleType">
   <property name="text">
    <string>Edit &amp;vehicle types</string>
   </property>
  </action>
  <action name="actEditArticle">
   <property name="text">
    <string>Edit &amp;articles</string>
   </property>
  </action>
  <action name="actEditUnit">
   <property name="text">
    <string>Edit &amp;units</string>
   </property>
  </action>
  <action name="actEditConnector">
   <property name="text">
    <string>Edit &amp;connectors</string>
   </property>
  </action>
  <action name="actCreateTestConnectionCable">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actEditTestNode">
   <property name="text">
    <string>Test node edition</string>
   </property>
  </action>
  <action name="actConnectTestCable">
   <property name="text">
    <string>Connect test cable</string>
   </property>
  </action>
  <action name="actDisconnectTestCable">
   <property name="text">
    <string>&amp;Disconnect test cable</string>
   </property>
  </action>
  <action name="actEditTest">
   <property name="text">
    <string>&amp;Test edition</string>
   </property>
  </action>
  <action name="actEditTestItem">
   <property name="text">
    <string>Test &amp;item edition</string>
   </property>
  </action>
  <action name="actEditTestConnectionCable">
   <property name="text">
    <string>Test connection cable edition</string>
   </property>
  </action>
  <action name="actCableChecker">
   <property name="text">
    <string>&amp;Cable checker</string>
   </property>
  </action>
  <action name="actEditLinkBeam">
   <property name="text">
    <string>Edit link &amp;beams</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
